docs/azure-saas-docs/content/en/components/Identity/identity-framework-setup-manual.md

```diff
@@ -0,0 +1,392 @@
+---
+type: docs
+title: "Identity Framework Setup - Manual"
+weight: 5000
+toc_hide: true
+---
+
+On this page, you will find instructions for how to manually setup the Identity Framework through a combination of the Azure dashboard as well as relevant PowerShell commands
+
+# Setup Identity Framework - Manual (Advanced)
+
+## 1. Requirements
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/)
+- [Open SSL](https://www.openssl.org/)
+  - Note: An OpenSSL executable is also bundled with Git for Windows at `C:\Program Files\Git\usr\bin\openssl.exe`
+- Visual Studio Code
+   - Tools:
+   - [Azure AD B2C Custom Policy Extension](https://marketplace.visualstudio.com/items?itemName=AzureADB2CTools.aadb2c)
+
+## 2. Instructions
+
+### Step 1. Create B2C Tenant in Azure
+
+![(Azure Add Service Screenshot)](/azure-saas/images/identity-framework-manual-step-1-adding-service.png)
+
+- From the dashboard, click `Create a Resource` and select `Azure Active Directory B2C`
+- Note: A reference is registered in your subscription at time of creation to associate the tenant with your subscription.
+
+### Step 2. Log into newly created Tenant
+
+![(Switch Directory Screenshot)](/azure-saas/images/identity-framework-manual-step-2-switch-tenant-quicklink.png)
+
+- Upon being created, your Azure AD B2C Tenant will create a federated identity with your Azure subscription account.
```

Comment:
I'm not sure how I feel about this sentence. It might be hurting more than helping.

This is technically known as Azure AD B2B, which opens up a whole other can of worms of things to explain. Probably easier to just remove it and not explain what's happening under the hood here.